tests/unit_tests/db_engine_specs/test_gsheets.py

```diff
@@ -73,8 +70,8 @@ def test_validate_parameters_catalog(
         ProgrammingError("Unsupported table: https://www.google.com/"),
     ]
 
-    properties: GSheetsPropertiesType = {
-        "parameters": {
+    engine_params = {
```

Comment:
can we define properties here as having both parameters and extra/engine_params?